packages/idealo.design/src/server/server.js

```diff
@@ -123,13 +122,21 @@ if (CLIENT_ID) {
     done(null, user)
   }));
 }
+function isBasicAuthenticated(req, res, next){
+  const username =req.header('username')
+  const password =req.header('password')
+  if (username && password){
+    return next();
+  }
```

Comment:
careful - you let anybody import who HAS a username and password, but you're not checking agains the validity. 

Also, when axios sends the auth header properly, it will look like this:
`Authorization: basic YmxhOmJsdWJi`

(for username: bla, password: blubb)

So when decoding this, you can do sth like:

```javascript
const base64Credentials =  req.headers.authorization.split(' ')[1];
const credentials = Buffer.from(base64Credentials, 'base64').toString('ascii');
const [username, password] = credentials.split(':');
if (username === process.env.UPLOADER_USERNAME && password === process.env.UPLOADER_PWD) {
  // ... correct credentials...
}
```